extensions/devservices/keycloak/pom.xml

```diff
@@ -34,13 +34,7 @@
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.sun.activation</groupId>
-                    <artifactId>jakarta.activation</artifactId>
-                </exclusion>
-            </exclusions>
+            <artifactId>keycloak-client-common-synced</artifactId>
```

Comment:
This now brings the Apache HTTP client as a dependency.

We used to have:

```
[INFO] +- org.keycloak:keycloak-core:jar:25.0.6:compile
[INFO] |  +- org.keycloak:keycloak-common:jar:25.0.6:compile
[INFO] |  |  \- jakarta.activation:jakarta.activation-api:jar:2.1.3:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.18.1:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.18.1:compile
[INFO] |  +- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:jar:2.18.1:compile
[INFO] |  \- org.eclipse.microprofile.openapi:microprofile-openapi-api:jar:4.0.2:compile
```

we now have:

```
[INFO] +- org.keycloak:keycloak-client-common-synced:jar:26.0.2:compile
[INFO] |  +- org.jboss.logging:commons-logging-jboss-logging:jar:1.0.0.Final:runtime
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.14:compile
[INFO] |  |  \- org.apache.httpcomponents:httpcore:jar:4.4.16:compile
[INFO] |  +- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:jar:2.18.1:compile
[INFO] |  |  +- com.fasterxml.jackson.core:jackson-core:jar:2.18.1:compile
[INFO] |  |  \- com.fasterxml.jackson.core:jackson-databind:jar:2.18.1:compile
[INFO] |  +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.18.1:compile
[INFO] |  +- org.eclipse.microprofile.openapi:microprofile-openapi-api:jar:4.0.2:compile
[INFO] |  \- jakarta.activation:jakarta.activation-api:jar:2.1.3:compile
```

I'm not sure we want that?

Now I don't know if we actually want to use the Admin Client now (or later?) to initialize the container in some way?

Reply:
> Now I don't know if we actually want to use the Admin Client now (or later?) to initialize the container in some way?

We don't use the Admin Client in KC Dev Svc. We use there Vert.x Web Client. We only need DTOs from `org.keycloak.representations`.

Not sure if it is relevant, but KC Dev Svc is now KC Admin Client dependency, I don't know if above-mentioned tree was done when rebased on current main. It only happened lately.

Reply:
It's not relevant for the tree of one dependency, but it is relevant for context - if you do exclusions.

Reply:
@pskopek Hi Peter, can you please do another iteration and drop Apache HTTP client as noticed by Guillaume ? It can consequences the same way shipping JAXRS client could've,  for ex, Keyloak devservice uses Vertx WebClient

Reply:
@sberyozkin it is already done my last update of this PR. `org.keycloak:keycloak-client-common-synced:jar:26.0.3` has no dependency on Apache HTTP client.

Reply:
Yes current status here is that I need to do another check to make sure everything is fine but we should hopefully be all good.
I'll do that today.

Reply:
I confirm the issue is solved on my side, thanks!